sdk/appcenter/src/main/java/com/microsoft/appcenter/ingestion/OneCollectorIngestion.java

```diff
@@ -153,12 +155,16 @@ public void close() throws IOException {
         mHttpClient.close();
     }
 
+    /** Call template implementation for One Collector. */
```

Comment:
Can you try formatting the file to see if android studio will generate changes? Otherwise other people will generate unexpected changes.

Ideally we use commit dialog in Android Studio and check most of the actions so that it's covered at commit time.